packages/docs/src/pages/en/getting-started/installation.md

```diff
@@ -86,7 +86,11 @@ pnpm dev
 
 ## Using Nuxt 3
 
-[Nuxt](https://nuxt.com/) is an open-source framework that has helpful features to quickly get you started with developing a full-stack Vue app, such as file-based routing, SSR and component auto-imports. Nuxt is powered by Vite, so the steps to get Vuetify working in Nuxt 3 are quite similar to the manual steps described above.
+[Nuxt](https://nuxt.com/) is an open-source framework that has helpful features to quickly get you started with developing a full-stack Vue app, such as file-based routing, SSR and component auto-imports.
+
+### Manual setup
+
+Nuxt is powered by Vite, so the steps to get Vuetify working in Nuxt 3 are quite similar to [the manual steps described below](#existing-projects).
```

Comment:
Uhmm, that's not 100% true, it is powered by nitro (and  vite-node) and can be used with Vite or Webpack 5